### PR TITLE
docs: add company charter

### PR DESCRIPTION
**Pull request by tancnle**
- Add the first version of our company charter (as a result of the [workshop](https://miro.com/app/board/uXjVOHw0M_4=/))
- Update missing doc references on the README

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: docs/peopleops/onboarding/onboarding-process.md [hunk 1]
```diff
@@ -25,14 +25,9 @@ and self-learning, whilst also remaining as asynchronous as possible.
...
 
-- [ ] Thoth Tech values are a living document. It is documented, refined and
-      revised based on lessons learned in the course of doing business.
-      Familiarise youself with our [Values](#).
-- [ ] Thoth Tech mission is to create tools that enhance education outcomes by
-      empowering students, connecting them with tutors and facilitating
-      personalised learning experiences.
-- [ ] Thoth Tech has a team charter that helps to steer the ways we work. Please
-      have a look at our [Charter](#)
+- [ ] Thoth Tech charter helps to steer the ways we work and is a living document.
```
Comment (kats8) [inline docs/peopleops/onboarding/onboarding-process.md:28]: Looks good, @tancnle - can we just add a "the" to make it "The Thoth Tech charter"? :)

File: docs/company/charter.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# Charter
+
+Thoth Tech charter is important for steering our team in the right direction,
```
Comment (kats8) [inline docs/company/charter.md:3]: @tancnle and if we can just adjust this to "The Thoth Tech charter" also?

**Review by kats8 — COMMENTED** [review]
Looking great, @tancnle 👍 - I'm new to reviewing via GitHub, so hopefully my two edit suggestions were commented correctly. If not, let me know :)

File: docs/peopleops/onboarding/onboarding-process.md (hunk 1 above)
Comment (tancnle) [reply docs/peopleops/onboarding/onboarding-process.md:28]: Raised a follow-up issue https://github.com/thoth-tech/handbook/issues/6

File: docs/company/charter.md (hunk 2 above)
Comment (tancnle) [reply docs/company/charter.md:3]: Raised a follow-up issue https://github.com/thoth-tech/handbook/issues/6

**Comment by tancnle** [comment]
Thank you for reviewing @kats8. For visibility, I have raise a separate issue to address your feedback https://github.com/thoth-tech/handbook/issues/6.

**Pull request merged**